tests/migration_test.go

```diff
@@ -632,7 +628,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			disks := vmi.Spec.Domain.Devices.Disks
 			disks[len(disks)-1].Serial = secretDiskSerial
 
-			vmi = runVMIAndExpectLaunchIgnoreWarnings(vmi, 180)
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 180)
```

Comment:
I think you need to use `tests.RunVMIAndExpectLaunchIgnoreWarnings` instead. True for all others as well.

Reply:
Correct to maintain the same logic. But I don't believe we see failures related to this so I am tempted to use this stricter version. WDYT?

Reply:
I barely understand why we need to care about warnings in such a dynamic system. But as long as there are no flakes due to this, feel free to keep as is.

Reply:
Done. I think we should default to a version that captures warnings but this can be followed up.